src/estree.ts

```diff
@@ -5,16 +5,19 @@ export interface _Node {
   loc?: SourceLocation | null;
 }
 
-export interface SourceLocation {
-  source?: string | null;
+export interface Location {
   start: Position;
   end: Position;
 }
 
+export interface SourceLocation extends Location {
+  source?: string | null;
+}
+
 export interface Position {
-  /** >= 1 */
+  /** \>= 1 */
```

Comment:
Why add escape to a comment? Probably your editor/tooling?

Reply:
Yes, Visual Studio Code uses `>` as the quote character.

Reply:
Do you mean this is needed for TSDoc to generate doc?

Reply:
Tt's the correct syntax because TSDoc uses markdown.